addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2310,8 +2310,8 @@ def test_out_invoice_post_1(self):
                 {
                     **self.tax_line_vals_1,
                     'currency_id': self.other_currency.id,
-                    'amount_currency': -180.0,
-                    'credit': 90.0,
+                    'amount_currency': -200.0,
+                    'credit': 100.0,
```

Comment:
@qdp-odoo This test was created to ensute the manual edition of taxes were kept when posting the invoice.
However, william changed it in 16.0 by setting `-180.0` instead of `-200.0`. So now the test is testing it no longer works... 
https://github.com/odoo/odoo/commit/d8d47f9ff8554f4b39487fd2f13c153c7d6f958d#diff-68204a256c7f4d9deab2de17f3d8394b4d96757eaeef3dbb78701cf2882f35daR2249

Since my `_sync_tax_lines` bring more flexibility to decide if the recomputation of taxes is needed or not, I fixed this test by restoring its original purpose.

Reply:
that's confusing... why not having a test dedicated for that?